JS.md

```diff
@@ -166,6 +166,29 @@ Otherwise, this function performs synchronous compilation of the `BufferSource`:
   the validated `Ast.module`.
 * On failure, a new `WebAssembly.CompileError` is thrown.
 
+### `WebAssembly.Module.exports`
```

Comment:
Ah, because of the heading I hadn't noticed that these are actually meant to be methods on module _objects_ (I suppose?). We probably should not do that -- it is a kind of reflection mechanism, and the general trend for ES has been to introduce new reflection mechanisms as static methods only, for a number of reasons (see all the `Object.*` methods). To be in line with that, the API here should rather be `WebAssembly.Module.exports(moduleObj)` and similarly for imports.